src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -45,7 +46,7 @@
 					<template v-if="page === 0">
 						<SetConversationName v-model="conversationNameInput"
 							@click-enter="handleEnter" />
-						<NcCheckboxRadioSwitch :checked.sync="isPublic">
+						<NcCheckboxRadioSwitch :checked.sync="isPublic" v-if="publicRoomsEnabled">
```

Comment:
ESLint: Attribute "v-if" should go before ":checked.sync"

```suggestion
						<NcCheckboxRadioSwitch v-if="publicRoomsEnabled" :checked.sync="isPublic">
```